test/compile_errors.zig

```diff
@@ -138,6 +138,42 @@ pub fn addCases(ctx: *TestContext) !void {
         "tmp.zig:2:1: error: invalid character: '\\t'",
     });
 
+    {
+        const case = ctx.obj("multiline error messages", .{});
+        case.backend = .stage2;
+
+        case.addError(
+            \\comptime {
+            \\    @compileError("hello\nworld");
+            \\}
+        , &[_][]const u8{
+            \\tmp.zig:2:5: error: hello
+            \\                    world
+        });
+
+        case.addError(
+            \\comptime {
+            \\    @compileError(
+            \\        \\
+            \\        \\hello!
+            \\        \\I'm a multiline error message.
+            \\        \\I hope to be very useful!
+            \\        \\
+            \\        \\also I will leave this trailing newline here if you don't mind
+            \\        \\
+            \\    );
+            \\}
+        , &[_][]const u8{
+            \\tmp.zig:2:5: error:
+            \\                    hello!
+            \\                    I'm a multiline error message.
+            \\                    I hope to be very useful!
+            \\
+            \\                    also I will leave this trailing newline here if you don't mind
+            \\
+        });
+    }
+
```

Comment:
This doesn't pass for me locally so it likely won't work in CI/CD here either but it definitely works when I test it on my own. I think it has to do with it just not really being possible to run stage2 tests like this yet.
Should I just comment it out for now like the stage2 tests below?